tosca-samples/org/ystia/yorc/samples/job/noscheduler/topologies/chainedJobs/types.yaml

```diff
@@ -0,0 +1,71 @@
+tosca_definitions_version: alien_dsl_2_0_0
+
+metadata:
+  template_name: ChainedJobs
+  template_version: 0.1.0-SNAPSHOT
+  template_author: appmanager
+
+description: |
+  Chaining job executions

```

Comment:
Ok so the idea here is to chain execution of Jobs over different computes...
Maybe it could be interesting to have an example with one job producing a data consumed by another job WDYT?
Could be done by for instance producing a file or maybe result could be stored within a node attribute with a `get_operation_output` on the `run` operation.

Reply:
Yes, could be interesting !
It seems Yorc is not populating such attribute values at the moment.
Switching the Pull Request to Work In Progress to put in place such an example.

Reply:
Updated the example to add a job retrieving and printing the output of commands run by two previous jobs